### DISCUSSION TITLE
New plugin for LDAP Authentication - Can you test it ?

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [wintercms/winter#525] [Show and tell]
Hi!

I wrote a (quite simple) plugin for LDAP Authentication in Winter CMS.

If someone it's interested can try it and submit new issue or ideas to improve it ?

Thanks.
Luca

Link on Github:
https://github.com/lucacalcaterra/wn-ldapauth-plugin

**Comment by LukeTowers** [comment]
Feel free to submit it to the [awesome-wintercms](https://github.com/wintercms/awesome-wintercms) repo!
